docs: 12-Factor tutorial updates (#2263)
canonical/charmcraft: docs/tutorial/kubernetes-charm-django.rst

Pack the Django app into a rock
-------------------------------

> First, we'll need a ``rockcraft.yaml`` file. We'll take advantage of a

Reviewer: ```suggestion
First, we'll need a ``rockcraft.yaml`` project file. We'll take advantage of a
```

This should prime the reader that `rockcraft.yaml` = project file.

Once Rockcraft has finished packing the Django rock, the
terminal will respond with something similar to

> ``Packed django-hello-world_0.1_<architecture>.rock``. The name of the
> ``.rock`` file will reflect your system's architecture. After the initial

Reviewer: ```suggestion
``Packed django-hello-world_0.1_<architecture>.rock``. The file name
reflects your system's architecture. After the initial
```

The files will automatically be created in your working directory.

We will need to integrate our Django app to the PostgreSQL database,

> which means we need to declare a requirement in the charm project file.

Reviewer: ```suggestion
which means we must declare a requirement in the charm project file.
```

One less "need to".
